src/test/java/api/support/http/ResourceClient.java

```diff
@@ -295,6 +292,7 @@ public IndividualResource createAtSpecificLocation(Builder builder)
     return new IndividualResource(getResponse);
   }
 
+  //TODO It's not used. Shall we remove it?
```

Comment:
Can we create a JIRA to consider this instead of modifying the code? I think we should revert this to the master version or just clean it up now (though I don't think it is related to the work required by the story).

Reply:
If it isn't used, then please delete the code

Reply:
Removed